05/app.js

```diff
@@ -8,19 +8,34 @@ const stats = {
 };
 
 /* tutaj umieść swój kod */
+const paragraphs = document.querySelectorAll("p");
+for (let i = 0; i < paragraphs.length; i++) {
+paragraphs[i].addEventListener("click", function () {
+        if (paragraphs[i].dataset.id === "p1") {
+            stats.paragraphs.p1 = stats.paragraphs.p1 + 1;
+        }
+}
+)}
+const els = document.querySelectorAll("a[href='/dolor.html']");
+els.forEach(function (element) {
+    element.onclick = function() {
+        stats.links["/dolor.html"] = stats.links["/dolor.html"] + 1;
+        element.href = "#";
```

Comment:
Rozwiązanie, które napisałeś działa w konkretnym przypadku, potrzebujemy uniwersalngo rozwiązania.
Zwróć uwagę, że `href` oraz `data-id` posiadają różne wartości.
Postaraj się poprawić to rozwiązanie ;)